README.md

```diff
@@ -32,22 +33,231 @@ scheduled_task { 'Run Notepad':
+The following example sets the scheduled task to run at 0700 on the first day of the month every month:
+
+~~~puppet
+scheduled_task { 'Disk Cleanup Monthly First Day':
+  ensure  => 'present',
+  command => 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe -File "C:\\Scripts\Clear-DiskSpace.ps1"',
+  enabled => 'true',
+  trigger => [{
+    'schedule'         => 'monthly',
+    'start_time'       => '07:00',
+    'on'               => [1]        # Run every month on the first day of the month.
+  }],
+  user    => 'system',
+}
+~~~
+
+One problem with the monthly trigger above is that we can't guarantee that the cleanup script doesn't run during a work day.
+We can specify the trigger to run on the task on the _first saturday_ of the month instead of the first day:
+
+~~~puppet
+scheduled_task { 'Disk Cleanup Monthly First Saturday':
+  ensure  => 'present',
+  command => 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe -File "C:\\Scripts\Clear-DiskSpace.ps1"',
+  enabled => 'true',
+  trigger => [{
+    'schedule'       => 'monthly',
+    'start_time'     => '07:00',
+    'day_of_week'    => 'sat',     # Specify the day of the week to trigger on
+    'which_occurence => 'first'    # Specify which occurance to trigger on, up to fifth
+  }],
+  user    => 'system',
+}
+~~~
 
 ## Reference
 
 ### Provider
 
+* win32_taskscheduler: This provider manages scheduled tasks on Windows using the legacy API.
 * taskscheduler_api2: Adapts the Puppet scheduled_task resource to use the modern Version 2 API.
 
 ### Type
 
-* scheduled_task: See the [Puppet resource documentation](https://puppet.com/docs/puppet/latest/types/scheduled_task.html) for more information.
+#### scheduled_task
+
+Installs and manages Windows Scheduled Tasks.
+All attributes except `name`, `command`, and `trigger` are optional; see the description of the [`trigger`]() attribute for details on setting schedules.
+
+##### `name`
+
+The name assigned to the scheduled task.
+This will uniquely identify the task on the system.
+
+##### `ensure`
+
+The basic property that the resource should be in.
+
+Valid values are `present`, `absent`.
+
+##### `arguments`
+
+Any arguments or flags that should be passed to the command.
+Multiple arguments should be specified as a space-separated string.
+
+##### `command`
+
+The full path to the application to run, without any arguments.
+
+##### `enabled`
+
+Whether the triggers for this task should be enabled.
+This attribute affects every trigger for the task; triggers cannot be enabled or disabled individually.
+
+Valid values are `true`, `false`.
+
+##### `password`
+
+The password for the user specified in the 'user' attribute.
+This is only used if specifying a user other than 'SYSTEM'.
+Since there is no way to retrieve the password used to set the account information for a task, this parameter will not be used to determine if a scheduled task is in sync or not.
```

Comment:
Consider re-ordering this so that the effect that the user cares about is up front in the sentence, and the justification in the second half. Users skimming docs will often not read entire sentences if the reason they should continue reading it is not immediately apparent.